iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -609,7 +880,7 @@ TEST_F(PoshRuntime_test, CreatingNodeWithInvalidNameLeadsToTermination)
     m_runtime->createNode(nodeProperty);
```

Comment:
You could also rename `CreatingNodeWithInvalidNameLeadsToTermination` to `CreatingNodeWithInvalidNameLeadsToError`. Also for all other occurrences in this file.